antlr/src/main/antlr4/org/apache/iotdb/db/qp/strategy/SqlBase.g4

```diff
@@ -641,6 +640,7 @@ dateFormat
 constant
     : dateExpression
     | NaN
+    | SDT
```

Comment:
Maybe you need to add some parameter name of sdt?
And Please add some example in the docs.

Reply:
if SDT is a constant, I don't think it needs a keyword.